vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -53,6 +58,7 @@ class ScrollViewShadowNode : public ShadowNodeBase {
   bool m_dismissKeyboardOnDrag = false;
 
   std::shared_ptr<SIPEventHandler> m_SIPEventHandler;
+  winrt::weak_ref<winrt::DependencyObject> m_lastSelectedItem = nullptr;
```

Comment:
Are the changes in this file still needed?

Reply:
See comment above.